src/components/dashboard/header.module.scss

```diff
@@ -0,0 +1,74 @@
+@use "src/styles/vars" as *;
+
+.main {
+  width: 100%;
+
+  display: grid;
+  grid-template-columns: 10px 96px 1fr;
+  grid-template-rows: 48px 48px;
+  gap: 5px;
+  grid-template-areas:
+    ". avatar title"
+    ". avatar description";
+
+  @include large {
+    width: 40%;
+
+    grid-template-columns: 10px 128px 1fr;
+    grid-template-rows: 64px 64px;
+  }
+}
+
+.avatar {
+  grid-area: avatar;
+
+  width: 100%;
+  height: 100%;
+
+  border: 1px solid currentColor;
+  border-radius: 100%;
+
+  padding: 0;
+  margin: 0;
+
+  background-color: $primary;
+  overflow: hidden;
+
+  z-index: 10;
```

Comment:
We should probably define some set `z-index` values as SCSS variables, easier to work with as it requires less thought

Reply:
Seconded, I saw in another file a bunch of z-indexes defined.